### PR TITLE
ENH enable using upstream number of letters

### PR DESCRIPTION
**Pull request by beckermr**
This PR changes the import map handling to grab the indexing number of letters from lincfgraph. I also added news from my previous PR and this one.

### CODE REVIEW

File: depfinder/reports.py [hunk 1]
```diff
@@ -49,10 +49,19 @@
 
 
 @lru_cache()
-def _import_map_cache(import_first_two_letters):
+def _import_map_num_letters():
+    req = requests.get(
+        'https://raw.githubusercontent.com/regro/libcfgraph/master'
+        'import_maps_meta.json')
```
Comment (beckermr) [inline depfinder/reports.py:55]:
```suggestion
        '/import_maps_meta.json')
```

File: news/new_news.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+**Added:**
+
+* Added ``--custom-namespaces`` option for custom name space searching.
+
+**Changed:**
+
+* Changed handling of import maps to compute number of letters from upstream data.
```
Comment (beckermr) [inline news/new_news.rst:7]:
```suggestion
* Changed handling of import maps to compute number of letters from upstream data. 
```

**Comment by beckermr** [comment]
@CJ-Wright This one is ready to go!

**Comment by CJ-Wright** [comment]
Thank you very much!

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

**Pull request merged**